<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at http://www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>documents</class>
 <widget class="QWidget" name="documents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="_documents" native="true">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>175</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="XTreeWidget" name="_doc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="_newDoc">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_editDoc">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_viewDoc">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;View</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_attachDoc">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&amp;Attach</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_detachDoc">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&amp;Detach</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>70</width>
            <height>41</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
